lib/vagrant-service-manager/plugin_util.rb

```diff
@@ -111,6 +111,7 @@ def self.execute_and_exit_on_fail(machine, ui, command)
 
       def self.execute_once(machine, ui, command)
         machine.communicate.sudo(command) do |_, data|
+          next if data.chomp.empty?
```

Comment:
It seems we are missing the root cause here. No way that this gets executed twice by design. Either there is a bug in Vagrant or we are missing something else.

If there is indeed something we need to workaround we need a better patch than this. What if for some reason we need another communicator call somewhere else? How would one kn ow/remember to add this _next_ statement. If there is a workaround needed we need to provide a way to apply this workaround to all potential uses of the communicator, for example by adding our own helper/utility method around the communicator here. 

But to start with we need to understand the problem first.